src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -151,36 +180,24 @@ function ChplListingEditUploadPage({ id }) {
           </Typography>
         </Container>
       </div>
-      <Container maxWidth="lg" id="main-content" tabIndex="-1">
+      <Container maxWidth="xl" id="main-content" tabIndex="-1">
+        <Box mt={8}>
+          <ListingContext.Provider value={newListingState}>
+            <ChplUploadListing
+              id={listing.id}
+              setErrors={setErrors}
+              setWarnings={setWarnings}
+            />
+          </ListingContext.Provider>
+        </Box>
         <div className={classes.container}>
-          <div>
-            <ListingContext.Provider value={newListingState}>
-              <ChplUploadListing
-                id={listing.id}
-                setErrors={setErrors}
-                setWarnings={setWarnings}
-              />
-            </ListingContext.Provider>
-          </div>
-          <div>
-            Difference(s)
-            <ul className="list-unstyled">
-              { diff.map((change) => (
-                <li key={change} dangerouslySetInnerHTML={{ __html: `${change}` }} />
-              ))}
-            </ul>
-          </div>
-          <div>
-            Current Listing
-          </div>
-          <div>
-            Updated Listing
-          </div>
-          <div>
+          <Box>
+            <Typography gutterBottom component={"h3"} style={{ fontWeight: '600' }} variant='h4'>Current Listing</Typography>
```

Comment:
The component says `h3`, but the variant says `h4`. Is that right?

Reply:
Correct,  its for the element to read as h3 for 508 but then styled like an h4